tests/database/odbc-w-enclave/msodbcsql.h

```diff
@@ -0,0 +1,730 @@
+#ifndef __msodbcsql_h__
+#define __msodbcsql_h__
+
+/*
+ *-----------------------------------------------------------------------------
+ * File:        msodbcsql.h
+ *
+ * Copyright:   Copyright (c) Microsoft Corporation
```

Comment:
What's the license of this file? Why does it have to be copied to the repo?

Reply:
SQL AEv2 feature is still in public preview. The release version of the latest ODBC header will not support all the features, so we have to modify the release version of ODBC header and copy it to app code

Reply:
Then there *must* be a big fat TODO somewhere that says exactly that and it should also say that this custom patched header can be removed once it's in GA.

Reply:
Will it be better to put in this header file or in the README file?

Reply:
Probably the header file.